pkg/scheduler/testing/wrappers.go

```diff
@@ -340,6 +340,17 @@ func (p *PodWrapper) Resources(resources v1.ResourceRequirements) *PodWrapper {
 	return p
 }
 
+func (p *PodWrapper) NodeAffinity(nodeAffinity *v1.NodeAffinity) *PodWrapper {
+	if p.Spec.Affinity == nil {
+		p.Spec.Affinity = &v1.Affinity{}
+	}
+	if p.Spec.Affinity.NodeAffinity == nil {
+		p.Spec.Affinity.NodeAffinity = &v1.NodeAffinity{}
+	}
+	p.Spec.Affinity.NodeAffinity = nodeAffinity
```

Comment:
This could be simplified to 
```suggestion
	p.Spec.Affinity.NodeAffinity = nodeAffinity
```
?